### PR TITLE
K8 s project files

### PR DESCRIPTION
**Pull request by JFer11**

### CODE REVIEW

File: k8s-resources/aws-eks-production-cluster/05-secret-backend.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  namespace: main-project
+  name: backend-secrets
+type: Opaque
+data:
+  SECRET_COOKIE_PASSWORD: dkRXd0xvTkNzMnhDR0I0UUZCdHlQdmtYM2pKaUVYNjk=
+  ACCESS_TOKEN_SECRET: dHJpcGxlag==
+  JWT_PRIVATE_KEY: anVzdGFub3RoZXJwcml2YXRla2V5
+  JWT_PUBLIC_KEY: bm90aW1wb3J0YW50
+  MOOSEND_API_KEY: NzBlNmQ1YWItM2UxZS00YjVhLWJhYjEtMDY3NzJjNTcyZDFm
+  NODEMAILER_PASSWORD: dWxwcW9kdG15dHBpd3RpaA==
```
Comment (jopealgorta) [inline k8s-resources/aws-eks-production-cluster/05-secret-backend.yaml:20]: All these passwords shouldn't be here. Ideally we will retrieve them from the AWS Secrets manager. I know that exists some terraform modules that do this.

File: k8s-resources/aws-eks-production-cluster/07-deploymentAndService-backend.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        app: nestjs
+    spec:
+      containers:
+        - name: nestjs
+          image: jfer11/thesis-backend:latest
+          ports:
+            - containerPort: 5000
+          imagePullPolicy: Always
+          command: ["/bin/sh", "-c"]
+          args: ["sleep 5; npm ci --ignore-script=prepare && prisma generate && prisma migrate deploy && npm start"]
```
Comment (jstaricco10) [inline k8s-resources/aws-eks-production-cluster/07-deploymentAndService-backend.yaml:23]:
```suggestion
          args: ["sleep 5; npm ci --ignore-script=prepare && prisma generate && prisma migrate deploy && npm run start:prod"]
```

File: k8s-resources/aws-eks-production-cluster/09-ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    - http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: backend-service
+                port:
+                  number: 5001
+#   - host: TOBECHANGED  # Cambiar esto por lo que devuelva: kubectl get ingress
```
Comment (jstaricco10) [inline k8s-resources/aws-eks-production-cluster/09-ingress.yaml:21]: Should this comment be here? Also comment the code in english

File: k8s-resources/local-minikube-cluster/04-deploymentAndService-backend.yaml [hunk 4]
```diff
@@ -20,10 +20,11 @@ spec:
             - containerPort: 5000
           imagePullPolicy: Always
           command: ["/bin/sh","-c"]
-          args: ["sleep 5; npm install && prisma generate && prisma migrate dev --name init && npm run start:dev"]
+          # args: ["sleep 5; npm install && prisma generate && prisma migrate dev --name init && npm run start:dev"]
+          args: ["sleep 5; npm ci --ignore-script=prepare && prisma generate && prisma migrate deploy && npm start"]
```
Comment (jstaricco10) [inline k8s-resources/local-minikube-cluster/04-deploymentAndService-backend.yaml:24]:
```suggestion
          args: ["sleep 5; npm ci --ignore-script=prepare && prisma generate && prisma migrate deploy && npm run start:prod"]
```

**Comment by jstaricco10** [comment]
Topi treis PRs com mais conflictos

**Comment by JFer11** [comment]
I will deal with all these comments in the next PR. I need to have merge up to date.

**Pull request merged**